From 49d07de375985a43dacf168273a6f8c9451a72c6 Mon Sep 17 00:00:00 2001
From: AndroPlus <mail@andro.plus>
Date: Sun, 30 Dec 2018 13:11:57 +0900
Subject: [PATCH 1/1] Disable Google Camera

Signed-off-by: AndroPlus <mail@andro.plus>
---
 googleapps.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/googleapps.mk b/googleapps.mk
index bf0c87f..8f7d10d 100644
--- a/googleapps.mk
+++ b/googleapps.mk
@@ -20,7 +20,6 @@ PRODUCT_PACKAGES += \
     CarrierMetrics \
     ConnectivityMonitor \
     FaceLock \
-    GoogleCamera \
     GoogleContacts \
     GoogleContactsSyncAdapter \
     GoogleExtShared \
-- 
2.17.1

