From 14bfdd2d26585547de4268098a8eb0a7f732e07a Mon Sep 17 00:00:00 2001
From: AndroPlus <mail@andro.plus>
Date: Sat, 30 Mar 2019 16:32:59 +0900
Subject: [PATCH 1/1] HACK to generate changelog

Signed-off-by: AndroPlus <mail@andro.plus>
---
 core/Makefile | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/Makefile b/core/Makefile
index 7fcaaf17d..13ae1f55f 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -861,6 +861,10 @@ INSTALLED_BOOTIMAGE_TARGET :=
 endif # BOARD_PREBUILT_BOOTIMAGE
 endif # TARGET_NO_KERNEL
 
+$(hide) ./vendor/lineage/build/tools/changelog.sh
+$(hide) mv $(PRODUCT_OUT)/Changelog.txt $(PRODUCT_OUT)/changelog_$(TARGET_DEVICE).txt
+
 # -----------------------------------------------------------------
 # NOTICE files
 #
-- 
2.17.1

